src/screens/transaction_details/components/msg/unknown/index.tsx

```diff
@@ -11,7 +11,7 @@ const Unknown = (props: {
   return (
     <pre className={classes.root}>
       <code>
-        {JSON.stringify(message.json, null, '\t')}
+        {JSON.stringify(message?.json, null, '\t')}
```

Comment:
please return to `{JSON.stringify(message.json, null, '\t')}` as message.json is a required key in the big dipper documentation

Reply:
Updated!